submarine-client/src/main/java/org/apache/submarine/client/cli/param/runjob/RunJobParameters.java

```diff
@@ -347,7 +347,7 @@ public boolean isDistributed() {
   @VisibleForTesting
   public static class UnderscoreConverterPropertyUtils extends PropertyUtils {
     @Override
-    public Property getProperty(Class<? extends Object> type, String name) throws IntrospectionException {
```

Comment:
Snakeyaml doesn't throw exception on getProperty since [version 1.19](https://www.javadoc.io/doc/org.yaml/snakeyaml/1.19/org/yaml/snakeyaml/constructor/Constructor.ConstructMapping.html)

Reply:
Thank you @aeioulisa for providing it, no block in my side.
I'm +1 for this PR.

Reply:
We need to be careful about binary compatibility for public APIs, for they could throw NoSuchMethodException during runtime, or cause compilation error during compile time.

However, this particular case is fine:
https://stackoverflow.com/questions/14734345/changing-the-interface-without-recompiling-the-implementing-class
`Changes to the throws clause of methods or constructors do not break compatibility with pre-existing binaries; these clauses are checked only at compile time.`

Reply:
> We need to be careful about binary compatibility for public APIs, for they could throw NoSuchMethodException during runtime, or cause compilation error during compile time.

Thanks @jojochuang for the catch, I agree with it.